backend/alanda-openapi-mixin/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.alanda</groupId>
+    <artifactId>alanda</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.alanda</groupId>
+  <artifactId>alanda-openapi-mixin</artifactId>
+  <version>1.0.0</version>
+  <packaging>pom</packaging>
+
+  <description>
+    Mixin to bundle dependencies, opinionated for using the openapi generator to create a spring (boot) server stub.
+    Import with scope 'pom' to get all dependencies, then use com.github.odavid.maven.plugins:mixin-maven-plugin
+    to embed the build:plugin section of this mixin and enable the plugins via properties below, with the form:
+    [plugin-name].skip false and name the .yaml/json file to generate the code from and name a generator to select
+    what to create.
+  </description>
+
+  <distributionManagement>
+    <repository>
+      <id>alanda-repo</id>
+      <url>https://repo.alanda.io/repository/maven-releases/</url>
+      <layout>default</layout>
+    </repository>
+    <snapshotRepository>
+      <id>alanda-repo</id>
+      <url>https://repo.alanda.io/repository/maven-snapshots/</url>
+      <layout>default</layout>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <properties>
+    <openapi.generator.plugin.skip>true</openapi.generator.plugin.skip>
+    <openapi.generator.name>forExmapleJavaForClientOrSpringForServerStub</openapi.generator.name>
```

Comment:
why is there placeholder code in here

Reply:
because this really needs to be overwritten by the importing person. it will not be triggered but serves as an example setup. you can also ignore this section even and set your own openapigenerator execution. That is possible, because we have the dependencies in any case. thats also the style that I have chosen for the project

Reply:
So in my actual project i can overwrite these by simply creating a <properties> and inserting this exact property (with a different value) again?

Reply:
yes :)

Reply:
@flwal-alanda all honor goes to him :D

Reply:
Ok, i now understand.  Thank you for elaborating